homeassistant/components/http/auth.py

```diff
@@ -29,19 +29,9 @@ def async_sign_path(hass, refresh_token_id, path, expiration):
         secret = hass.data[DATA_SIGN_SECRET] = secrets.token_hex()
 
     now = dt_util.utcnow()
-    return "{}?{}={}".format(
-        path,
-        SIGN_QUERY_PARAM,
-        jwt.encode(
-            {
-                "iss": refresh_token_id,
-                "path": path,
-                "iat": now,
-                "exp": now + expiration,
-            },
-            secret,
-            algorithm="HS256",
-        ).decode(),
+    return (
+        f"{path}?{SIGN_QUERY_PARAM}="
+        f"{jwt.encode({'iss': refresh_token_id,'path': path,'iat': now,'exp': now + expiration,},secret,algorithm='HS256',).decode()}"
```

Comment:
This seems much harder to read. Wouldn't it be better to assign the output of the encode to a temporary variable and then f-string with that?

Reply:
Yes, indeed. I have used one now to make it easier to read.